tests/examples/source/devfiles/nodejs/devfile-with-multiple-defaults.yaml

```diff
@@ -51,5 +51,27 @@ commands:
       component: runtime
       commandLine: npm start
       workingDir: ${CHE_PROJECTS_ROOT}/nodejs-starter
+      group:
+        kind: run
+  - exec:
+      id: singleenv
+      component: runtime
+      commandLine: "mkdir $ENV1"
+      env:
+       - name: ENV1
+         value: "test_env_variable"
+      workingDir: ${CHE_PROJECTS_ROOT}/nodejs-starter
+      group:
+        kind: run
+  - exec:
+      id: multipleenv
+      component: runtime
+      commandLine: "mkdir $ENV1 $ENV2"
+      env:
+       - name: ENV1
+         value: "test_env_variable1"
+       - name: ENV2
+         value: "test_env_variable2"
```

Comment:
can we have a env var with a space too?

Reply:
added test case with env variables with spaces.